test/cpp/api/fft.cpp

```diff
@@ -5,12 +5,123 @@
 
 #include <test/cpp/api/support.h>
 
+// Naive DFT of a 1 dimensional tensor
```

Comment:
These tests are super cool and we should definitely keep them, but in general let's bias towards Python testing where it's possible and reserve future C++ tests for testing what we can't test from Python.